tests/unit_test.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+import numpy as np
+
+import MCHammer
+
+
+def test_sampler():
```

Comment:
Add a docstring to the function.
```suggestion
def test_sampler():
    '''
    Test that the sampler recovers a multi-variate Gaussian distribution with approximately the correct standard deviation.
    '''
```